### PR TITLE
Add activities for status change [SCI-4827]

### PR DESCRIPTION
**Pull request by urbanrotnik**
Jira ticket: [SCI-4827](https://biosistemika.atlassian.net/browse/SCI-4827)

PS: 
Check GitLab: https://gitlab.com/biosistemika/scinote/addons/electronic_signatures/-/merge_requests/94

### CODE REVIEW

File: config/initializers/extends.rb [hunk 1]
```diff
@@ -287,13 +288,18 @@ class Extends
     archive_inventory_item: 142,
     restore_inventory_item: 143,
     archive_inventory: 144,
-    restore_inventory: 145
+    restore_inventory: 145,
+    change_status_on_task_flow: 146,
+    sign_task: 147,
+    cosign_task: 148,
```
Comment (aignatov-bio) [inline config/initializers/extends.rb:294]: Why it here?

**Review by aignatov-bio — CHANGES_REQUESTED** [review]

File: app/controllers/my_modules_controller.rb [hunk 2]
```diff
@@ -256,7 +256,12 @@ def update_state
     new_status = @my_module.my_module_status_flow.my_module_statuses.find_by(id: update_status_params[:status_id])
     return render_404 unless new_status
 
+    old_status = @my_module.my_module_status
     @my_module.update(my_module_status: new_status)
+    new_status = @my_module.my_module_status
```
Comment (radiokills) [inline app/controllers/my_modules_controller.rb:261]: Do you really need this? Don't you have new status already?

File: config/initializers/extends.rb [hunk 3]
```diff
@@ -288,13 +291,15 @@ class Extends
...
+    export_inventory_snapshot_items_assigned_to_task: 147,
+    change_status_on_task_flow: 148 # 149, 150, 151 in AdddOn!
   }
 
   ACTIVITY_GROUPS = {
     projects: [*0..7, 32, 33, 34, 95, 108, 65, 109],
     task_results: [23, 26, 25, 42, 24, 40, 41, 99, 110, 122, 116, 128],
-    task: [8, 58, 9, 59, *10..14, 35, 36, 37, 53, 54, *60..63, 138, 139, 140, 64, *66..69, 106, 126, 120, 132],
+    task: [8, 58, 9, 59, *10..14, 35, 36, 37, 53, 54, *60..63, 138, 139, 140, 64, 66, 106, 126, 120, 132,
+           *146..148],
```
Comment (urbanrotnik) [inline config/initializers/extends.rb:302]: @okriuchykhin  Should be `146` and `147` included here?

File: app/controllers/my_modules_controller.rb [hunk 4]
```diff
@@ -266,13 +266,15 @@ def update_state
     new_status = @my_module.my_module_status_flow.my_module_statuses.find_by(id: update_status_params[:status_id])
     return render_404 unless new_status
 
+    old_status = @my_module.my_module_status
```
Comment (aignatov-bio) [inline app/controllers/my_modules_controller.rb:269]: Just store id

**Review by aignatov-bio — CHANGES_REQUESTED** [review]

**Review by aignatov-bio — APPROVED** [review]

**Review by aignatov-bio — APPROVED** [review]

**Pull request merged**